lib/glific/partners/billing.ex

```diff
@@ -275,12 +290,12 @@ defmodule Glific.Partners.Billing do
          {:ok, _customer} <-
            Stripe.Customer.update(billing.stripe_customer_id, %{
              invoice_settings: %{default_payment_method: stripe_payment_method_id}
-           }) do
-      update_billing(billing, %{stripe_payment_method_id: stripe_payment_method_id})
-      |> case do
-        {:ok, billing} -> {:ok, billing}
-        {:error, _} -> {:error, %{message: "Error while saving details"}}
-      end
+           }),
+         {:ok, billing} <-
+           update_billing(billing, %{stripe_payment_method_id: stripe_payment_method_id}) do
+      {:ok, billing}
+    else
+      _ -> {:error, %{message: "Error while saving details"}}
```

Comment:
Let's put a {:error, _} clause to get the sepcific error message else send the complete response.